server/.babelrc

```diff
@@ -1,24 +1,19 @@
 {
   "presets": [
     "@babel/preset-react",
-    "@babel/preset-typescript",
     [
       "@babel/preset-env",
       {
-        "corejs": {
```

Comment:
corejs polyfills for reflect metadata were conflicting with `reflect-metadata` so the easiest thing to do here was remove.